packages/catalog-model/src/schema/kinds/Environment.v1alpha1.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "EnvironmentV1alpha1",
+  "description": "An Environment represents a single static location that software is deployed to",
+  "examples": [
+    {
+      "apiVersion": "backstage.io/v1alpha1",
+      "kind": "Environment",
+      "metadata": {
+        "name": "artists",
+        "description": "Everything about artists"
+      },
+      "spec": {
+        "owner": "artist-relations-team"
```

Comment:
Perhaps `prod-ops-team` or `qa-team`?

Reply:
this was a holdover from the system that I copied this example from 😓 . Not sure `Environment` should have an owner,  and it's not in the initial spec. Removing for now.